articles/machines/qc-trace-simulator/depth-counter.md

```diff
@@ -13,16 +13,16 @@ The `Depth Counter` is a part of the quantum computer [Trace
 Simulator](xref:microsoft.quantum.machines.qc-trace-simulator.intro).
 It is used to gather counts of the depth of
 every operation invoked in a quantum program. All operations from
-<xref:microsoft.quantum.primitive> are expressed in terms of single qubit rotations,
-T gates, single qubit Clifford gates, CNOT gates and measurements of multi-qubit
-Pauli observables. Users can set the depth for each of the primitive operations via the `gateTimes` field of <xref:microsoft.quantum.simulation.simulators.qctracesimulators.qctracesimulatorconfiguration>.
+<xref:microsoft.quantum.intrinsic> are expressed in terms of single qubit rotations,
+T gates, single qubit Clifford gates, CNOT gates, and measurements of multi-qubit
+Pauli observables. Users can set the depth for each of the primitive operations via the `gateTimes` field of <xref:microsoft.quantum.simulation.simulators.qctraceSimulators.qctracesimulatorconfiguration>.
```

Comment:
The Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.QCTraceSimulatorConfiguration name refers to a C# symbol rather than a Q# symbol, and thus should be formatted as `<xref:Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.QCTraceSimulatorConfiguration>` (in particular, only Q# names get transformed to lowercase, due to the aforementioned issue).
It may well be, though, that cross-references to C# still aren't working; we were still having trouble with `<xref:System.IDisposable>`, for instance, despite that being a completely valid xref.
Do you know if there was any progress with getting cross-references to C# documentation fixed?

Reply:
@cgranade, there is some correspondence with @cpalmer2020.  I'm willing to pick up the conversation, but we'll likely get more response if you or Cathy pursue it.

Reply:
@cgranade thanks for clarification of casing on C# vs Q#.